### ISSUE TITLE
No datasource out-of-the-box

### ISSUE THREAD

**Issue by TBBle** [cyber-dojo/grafana#1]
The [README.md]'s blank, so I assume this is still WIP?

Anyway, I have a Dashboard but no data source. I hoped 2b8f355 would give me a clue, but I couldn't work out if things were supposed to work with port 4557 on the app_default network address for cyber-dojo-runner.

**Comment by JonJagger** [comment]
Yes it's not 100% finished. The main thing unfinished is a way to make the data-source permanent.
There is a prometheus service (see commander's docker-compose.yml) which Grafana scrapes from. 
The prometheus.yml file in the prometheus github repo shows the underlying services and ports (eg runner:4557). From memory the Grafana data source is prometheus on 9090.

**Comment by TBBle** [comment]
Okay, I got it working. For anyone else who wanders along between now and when it's out-of-the-box setup, I set the name to `${DS_PROMETHEUS}`, the Url to http://cyber-dojo-prometheus:9090/ and the Access to `proxy`.
